### ISSUE TITLE
typescript-resolvers plugins + import-types preset

### ISSUE THREAD

**Issue by VienNguyen113** [dotansimha/graphql-code-generator#3332]
Can we use the import-types preset and type typescript-resolvers plugins together?
here is my config for generating types file 
```
 ./graphql-types.ts :
    plugins:
      - typescript
      - typescript-operations
```
here is the config for generating resolver types 
```
./graphql-resolvers.ts:
    preset: import-types
    presetConfig:
     typesPath: ./graphql-types
    plugins:
      - typescript-resolvers
    config:
      immutableTypes: true
      preResolveTypes: true
```
and 
<img width="744" alt="Screen Shot 2020-01-21 at 15 18 13" src="https://user-images.githubusercontent.com/11764707/72786853-40825900-3c61-11ea-95ad-2bd9a887ef73.png">
<img width="744" alt="Screen Shot 2020-01-21 at 15 18 31" src="https://user-images.githubusercontent.com/11764707/72786875-4b3cee00-3c61-11ea-8fd3-53366c3c5ad3.png">

- Current Error. Imported Types from generated files but never used.

- Expected Results: It can use types from imported Types (example Types.Maybe ..etc)

**Comment by dotansimha** [comment]
It's not supported yet. PRs are welcome :)

**Comment by charlex** [comment]
I am interested in this as well. Let me know if I can help!

**Comment by ardatan** [comment]
@charlex You can help and contribute :) PRs are welcome as @dotansimha said.

**Comment by dotansimha** [comment]
@VienNguyen113 I added support for this in: https://github.com/dotansimha/graphql-code-generator/pull/4289 

It's general support for `namespacedImportName` configuration key in `typescript-resovlers`, which presets are using as well. So now if this plugin will get `namespacedImportName` configuration, it will use it correctly.

**Comment by dotansimha** [comment]
Fixed in `v1.16.0`
